app/assets/javascripts/client/about/about_sidebar.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar about-sidebar">
+  <div class="sidebar-section">
+    <h2>Share this page</h2>
+    <p>Copy this link to share this page.</p>
+    <div class="input-group">
+      <input id="share-url" value="{{aboutSidebar.shareURL}}" class="form-control">
+      <div class="input-group-addon" data-clipboard-target="#share-url"><i class="fa fa-copy"></i></div>
+    </div>
+  </div>
+  <div class="sidebar-section">
+    <h2>PDredesign</h2>
+    <p><a ui-sref="about.pdredesign">About PDredesign</a></p>
+  </div>
+  <div class="sidebar-section">
+    <h2>1. Readiness Assessment</h2>
+    <p><a ui-sref="about.ra-assessment">About Readiness Assessment</a></p>
+    <p><a ui-sref="faqs({topic: 'Readiness Assessment'})">FAQs</a></p>
+    <p><a ng-click="aboutSidebar.openAssessmentModal()">Facilitate New Assessment</a></p>
+    <p><a ui-sref="assessments">View Assessment List</a></p>
```

Comment:
The current behaviour is that you're redirected to the login page, presumably because of `authenticate: true` in the routes. I didn't change any of that.